content/systems/filecoin_vm/sysactors/init_actor.md

```diff
@@ -2,12 +2,16 @@
 title: InitActor
 weight: 1
 dashboardWeight: 2
-dashboardState: incomplete
+dashboardState: stable
 dashboardAudit: 0
 dashboardTests: 0
 ---
 
 # InitActor
 ---
 
-{{<embed src="/modules/actors/builtin/init/init_actor.go" lang="go">}}
+The `InitActor` has the power to create new actors, e.g., those that enter the system. It maintains a table resolving a public key and temporary actor addresses to their canonical ID-addresses. Invalid CIDs should not get committed to the state tree.
+
+The actor should be able to construct a canonical ID address for the actor that will persist even in case of a chain re-organisation.
```

Comment:
The canonical ID address does _not_ persist even in case of chain re-org. The address types are distinguished by this:
- the ID address is canonical but does not survive reorg
- the actor or pubkey address survives reorgs